lib/libmarshal.map

```diff
@@ -212,8 +210,6 @@
         Tss2_MU_TPMU_SIGNATURE_Unmarshal;
         Tss2_MU_TPMU_SENSITIVE_COMPOSITE_Marshal;
         Tss2_MU_TPMU_SENSITIVE_COMPOSITE_Unmarshal;
-        Tss2_MU_TPMU_ENCRYPTED_SECRET_Marshal;
-        Tss2_MU_TPMU_ENCRYPTED_SECRET_Unmarshal;
```

Comment:
We shouldn't remove these from here. They are still in the marshaling spec.

Reply:
Sorry, there were duplicates inside the file...
Updated commit message...